.github/workflows/build-manylinux-container-images.yml

```
@@ -48,6 +48,7 @@ jobs:
         YEAR:
         - _2_24
         - _2_28
+        - _2_34
```

Match the change note.
```suggestion
        - _2_32
```